webapp/client/src/lib/translations.js

```diff
@@ -273,6 +273,40 @@ const translations = {
         "Mit Ihrer Registrierung beantragen Sie einen Freischaltcode bei Ihrer Finanzverwaltung. Sie erhalten diesen mit einem Brief <bold>innerhalb von zwei Wochen</bold> nach erfolgreicher Beantragung. Wenn Sie die Zusammenveranlagung nutzen möchten, reicht es aus, wenn sich eine Person registriert.",
     },
   },
+  filing: {
+    success: {
+      title:
+        "Ihre Informationen wurden erfolgreich verschickt. Speichern Sie Ihre Nachweise für Ihre Unterlagen.",
+      intro:
+        "Ihre Informationen wurden erfolgreich an Ihre Finanzverwaltung übermittelt. Bewahren Sie Ihre Nachweise für Nachfragen gut auf.",
+      transfer_ticket: {
+        heading: "Transferticket",
+        text: "Mit dem Transferticket wird die erfolgreiche Übermittlung Ihrer Daten durch die Finanzverwaltung bestätigt. Notieren Sie sich Ihr Transferticket, damit Sie nachweisen können, dass Ihre Daten erfolgreich übermittelt wurden.",
+        your: "Ihr Transferticket:",
```

Comment:
Maybe we could rename that because I find the key quite hard to understand. What about `sub_headline

Reply:
What do you mean? We have the title and intro of the success case and then we have two parts of the page:

- transfer_ticket
- pdf

Both of them have a heading and a text, and then another specific text

Reply:
Yes, I should have commented on the correct lines. I meant 'your'

Reply:
Okey. What about "your_heading" ? Because "heading" is "Transferticket", then "your_heading" is "Ihr Transferticket"?